src/Pos/Explorer/Txp/Local.hs

```diff
@@ -130,9 +136,9 @@ eTxProcessTransaction itw@(txId, TxAux {taTx = UnsafeTx {..}}) = do
                      flip evalUtxoStateT resolved $
                      flip runReaderT eet $
                      runExplorerReaderWrapper $
+                     flip runReaderT genStks $
```

Comment:
@volhovM I tried to do it, but there is not `HasLens` for first and second elements of pair